app/src/main/java/org/p2p/wallet/newsend/NewSendPresenter.kt

```diff
@@ -247,13 +261,17 @@ class NewSendPresenter(
         val currentState = feeRelayerManager.getState()
         if (currentState !is FeeRelayerState.UpdateFee) return
 
-        val solanaFee = currentState.solanaFee
-        if (solanaFee == null) {
+        if (inputAmount == emptyString()) {
```

Comment:
it's wrong. 
the user can waste all 100 free transactions and not enter anything.
we should show free transactions only if user has them